### ISSUE TITLE
IENS is not substituted in manifest.json

### ISSUE THREAD

**Issue by berland** [equinor/ert#9556]
In the file `manifest.json`, it seems possible to end up not having e.g. `<IENS>` substituted with the realization number.

This yields error messages printed to the terminal:

```
2024-12-13 13:45:19,724 - ert.scheduler.job - ert_gui_simulation_thread - ERROR - Expected file /scratch/fmu/xxxx/01_drogon_ahm-xxxxx/realization-0/iter-0/eclipse/model/DROGON-<IENS>.UNSMRY not created by forward model!
Expected file /scratch/fmu/xxxxx/01_drogon_ahm-xxxxx/realization-0/iter-0/eclipse/model/DROGON-<IENS>.SMSPEC not created by forward model!
```

**Comment by berland** [comment]
More context: https://equinor.slack.com/archives/C01MCMWD075/p1734103442252469

**Comment by berland** [comment]
Solved in #9233. Error was obtained with Ert 11.1.9
